app/models/user.rb

```diff
@@ -8,4 +8,7 @@ class User < ApplicationRecord
 
   # add validations and rspec
   validates :nickname, presence: true, uniqueness: true
+
+  has_many :joined_groups, class_name: 'PlayerGroup', foreign_key: :joined_id
```

Comment:
the `joined_groups` should be the model `Group` not `PlayerGroup`
also, where is the `joined_id` coming from?